### PR TITLE
fix reverse option for cmvn

### PR DESCRIPTION
**Pull request by magictron**
The reverse operation order needs to be divide -> subtract (rather than subtract -> divide) w.r.t normalization operation order add + multiply

### CODE REVIEW

**Comment by kamo-naoyuki** [comment]
Oh, thanks.

**Comment by kamo-naoyuki** [comment]
Would you fix the CI error?

**Comment by magictron** [comment]
> Would you fix the CI error?

sorry that I am not that familiar with the process, I just edited the file from the web. Can you take it over?

Thanks

File: espnet/transform/cmvn.py [hunk 1]
```diff
@@ -106,11 +106,11 @@ def __call__(self, x, uttid=None):
             if self.norm_vars:
                 x = np.multiply(x, self.scale[spk])
 
-        else:
-            if self.norm_means:
-                x = np.subtract(x, self.bias[spk])
+        else:
```
Comment (kamo-naoyuki) [inline espnet/transform/cmvn.py:109]: Please delete the extra spaces.
```suggestion
        else:
```

**Pull request merged**